MeshGeoToolsLib/BoundaryElementsAtPoint.cpp

```diff
@@ -37,10 +37,19 @@ BoundaryElementsAtPoint::BoundaryElementsAtPoint(
             "one is expected.",
             node_ids.size(), _point[0], _point[1], _point[2]);
 
+    auto& mesh_nodes =
+        const_cast<std::vector<MeshLib::Node*>&>(_mesh.getNodes());
```

Comment:
Not sure here: Maybe the const cast is not necessary? It is only used in the identifyFace call through a pointer...

Reply:
`identifyFace` expects a non const array. As @chleh mentioned this is probably bad design in `identifyFace`.